colibri/src/globaldb.rs

```diff
@@ -62,6 +63,38 @@ impl GlobalDB {
                 })
         })
     }
+
+    /// Get all active RPC endpoints for a specific blockchain.
+    pub async fn get_rpc_nodes(&self, blockchain: SupportedBlockchain) -> Result<Vec<RpcNode>> {
+        let conn = self.conn.lock().await;
+        conn.prepare("SELECT name, endpoint FROM default_rpc_nodes WHERE blockchain=? AND name NOT LIKE '%etherscan%' AND active=1 AND (CAST(weight as decimal) != 0 OR owned == 1) ORDER BY name;")
+            .and_then(|mut stmt| {
+                let mut rows = stmt.query(rusqlite::params![blockchain.as_str()])?;
+                let mut nodes = Vec::new();
+                while let Some(row) = rows.next()? {
+                    let name: String = row.get(0)?;
+                    let endpoint: String = row.get(1)?;
+                    nodes.push(RpcNode {
+                        name,
+                        endpoint,
+                        blockchain,
+                    });
+                }
+                Ok(nodes)
+            })
+    }
+
+    pub async fn is_uniswap_v3_position(&self, asset_id: &str) -> Result<bool> {
+        let conn = self.conn.lock().await;
+        let mut stmt = conn.prepare(
+            "SELECT 1 FROM evm_tokens
+             WHERE identifier = ?
+             AND protocol = 'UNI-V3'
+             LIMIT 1"
+        )?;
```

Comment:
question: we do label them right? I didn't find any in my db itself

Reply:
Isaac confirmed it 
![image](https://github.com/user-attachments/assets/f3f62d69-fcbf-4815-8dff-121ad7d557f1)